## Análise estatística
### Estudo de caso 2

1. Suponha que o peso médio dos Pinguins Reis encontrados em uma colônia na Antártica no último ano foi 15.4 kg. Em uma amostra de 35 pinguins na mesma época neste ano, na mesma colônia, o peso médio foi 14.1kg. Assuma que o desvio padrão da população seja 2kg. A um nível de significância de 5%, nós podemos rejeitar a hipótese nula de que o peso médio dos pinguins não difere do último ano?
Hipóteses:
* 𝐻0: 𝜇0 = 15.4kg
* 𝐻1: 𝜇0 ≠ 15.4kg

In [1]:
# Estatística de Teste:
xbar = 14.1 # media da amostra
mu0 = 15.4 # valor da hipótese
sigma = 2 # desvio padrão da população
n = 35 # tamanho da amostra 
z  = (xbar-mu0)/(sigma/sqrt(n)) # estatística de teste
print(z)

[1] -3.845452


Agora os valores críticos em um nível de significância 𝛼 = 0.05

In [2]:
alpha = 0.05
z.half.alpha = qnorm(1 - alpha/2) # Valor crítico
print(c(-z.half.alpha,z.half.alpha))


[1] -1.959964  1.959964


A estatística de teste z = -3.845452 está fora dos valores críticos -1.96, 1.96, ou seja, está fora da região de aceitação. Portanto, a um nível de significância de 0.05, nós rejeitamos a hipótese nula de que o peso médio dos pinguins se manteve em relação ao último ano.

---


2. Na indústria cerâmica, avalia-se sistematicamente a resistência de amostras de massas cerâmicas, após o processo de queima. Dessas avaliações, sabe-se que certo tipo de massa tem resistência mecânica aproximadamente normal, com média 53 MPa e variância 16 MPa². Após a troca de alguns fornecedores de matérias-primas, deseja-se verificar se houve alteração na qualidade. Uma amostra de 15 corpos de prova de massa cerâmica acusou média igual a 50 MPa. Qual é a conclusão ao nível de significância de 5 %?
Hipóteses:
* 𝐻0: 𝜇0 = 53kg
* 𝐻1: 𝜇0 ≠ 53kg

In [3]:
# Estatística de Teste:
xbar = 50 # media da amostra
mu0 = 53 # valor da hipótese
sigma = 16 # desvio padrão da população
n = 15 # tamanho da amostra 
z  = sqrt(n)*(xbar-mu0)/sigma # estatística de teste
print(z)

[1] -0.7261844


In [4]:
alpha = 0.05
rc <- qnorm(alpha)
rc

[1] -1.644854

A estatística de teste z = -0.726 está fora dos valores críticos -1.64, isso significa que está fora da região de aceitação.
Ao aplicar o nível de significância de 0.05 (valor de Alpha apresentado), nós rejeitamos a hipótese nula de que a troca dos fornecedores não alterou a qualidade do produto.

---

3. O tempo para transmitir 10 MB em determinada rede de computadores varia segundo um modelo normal, com média 7,4 seg e variância 1,3 𝑠𝑒𝑔². Depois de algumas mudanças na rede acredita-se numa redução no tempo de transmissão de dados, além de uma possível alteração na variabilidade. Foram realizados 10 ensaios independentes com um arquivo de 10 MB e foram anotados os tempos de transmissão, em segundos:
6,8; 7,1; 5,9; 7,5; 6,3; 6,9; 7,2; 7,6; 6,6; 6,3
Existe evidência suficiente de que o tempo médio de transmissão foi reduzido? Use nível de significância de 1%.
Hipóteses:
* 𝐻0: 𝜇0 = 7.4𝑠
* 𝐻1: 𝜇0 < 7.4𝑠

In [5]:
# teste unilateral
alpha = 0.01
amostra <- c(6.8, 7.1, 5.9, 7.5, 6.3, 6.9, 7.2, 7.6, 6.6, 6.3)
n = length(amostra)# tamanho da amostra
df = n-1 # graus de liberdade
S = sqrt(1.3) # desvio padrão da 
mu0 = 7.4 # valor da hipótese
xbar <- mean(amostra) # média da amostra
tc <- qt(alpha/2,df=df) # Equação para calculo de TC
print(paste("tc: ",tc))
Tt = (sqrt(n)*(xbar - mu0))/S # Equação da Estatística de Teste
print(paste("Tt: ",Tt))

[1] "tc:  -3.24983554159213"
[1] "Tt:  -1.60863056905316"


A estatística de teste Tt = -1.6 está fora da região crítica. Ao aplicar o nível de significância de 1% nós aceitamos a hipótese de que a diferença foi significativa.